src/QsCompiler/QirGeneration/Context.cs

```diff
@@ -28,8 +28,10 @@ namespace Microsoft.Quantum.QsCompiler.QIR
     /// This class holds the shared state used across a QIR generation pass.
     /// It also holds a large number of shared utility routines.
     /// </summary>
+#pragma warning disable SA1404 // Code analysis suppression should have justification
     [SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1124:Do not use regions")]
```

Comment:
FYI, the intended way to avoid this warning is by adding a justification:

```suggestion
    [SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1124:Do not use regions", Justification = "Existing code already used regions.")]
```